### ISSUE TITLE
demo.py predict NO BBox on the 2 image in this repo

### ISSUE THREAD

**Issue by Dylanbron** [Tianxiaomo/pytorch-YOLOv4#75]
python3 ./demo.py ./cfg/yolov4-custom.cfg  ./weight/yolov4.pth ./data/dog.jpg
这是我用您给出的weight和cfg文件进行inference的命令，您看一下有什么问题吗？

顺便问一句，cfg/里面不同的配置文件的区别是什么，和weight文件有对应关系吗？需要配套使用吗？

**Comment by love2learn-7179** [comment]
@Dylanbron I think we need to use yolov4.weights instead of yolov4.pth for weightfile. For ./data/dog.jpg, I got below.
data/dog.jpg: Predicted in 0.207703 seconds.
truck: 0.925689
dog: 0.999999
bicycle: 1.000000
save plot results to predictions.jpg

**Comment by Dylanbron** [comment]
> @Dylanbron I think we need to use yolov4.weights instead of yolov4.pth for weightfile. For ./data/dog.jpg, I got below.
> 
> data/dog.jpg: Predicted in 0.207703 seconds.
> truck: 0.925689
> dog: 0.999999
> bicycle: 1.000000
> save plot results to predictions.jpg

Thanks for your reply.Can you  help me about how to get yolov4.weights? I wonder if I need to run the  darknet2pytorch.py to convert .pth file.
Hope for your help.

**Comment by Dylanbron** [comment]
Thank you! I find the weight file on the google drive

**Comment by Dylanbron** [comment]
Have you trained the model by yourself? I'm wondering how to get .weights file from checkpoints.pth.

**Comment by love2learn-7179** [comment]
yes, I fine-tuned the model little bit to get updated .pth, then I used the updated .pth in models.py for inference.

**Comment by Dylanbron** [comment]
Ok, I find that the model used in demo and evaluate_on_coco is darknet, and the model in train and model is Yolov4. Pretrained and  checkpoints (pth) file should be used in Yolov4 model. I hope that this info can be updated in README, there may be some people who have not noticed like   @Tianxiaomo

**Comment by yxwsfz** [comment]
> @Dylanbron I think we need to use yolov4.weights instead of yolov4.pth for weightfile. For ./data/dog.jpg, I got below.
> 
> data/dog.jpg: Predicted in 0.207703 seconds.
> truck: 0.925689
> dog: 0.999999
> bicycle: 1.000000
> save plot results to predictions.jpg

您好，我下载了作者提供的参数文件，想运行demo.py 设置了和你说的相同的参数，但是没有结果，我打印了一下中间过程，在model（image）的输出结果就是nan，所以没有输出结果，想问下是什么原因，除了demo.py还有什么其他的地方需要修改吗？
